boards/shields/st_b_cams_omv_mb1683/boards/stm32h7b3i_dk.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
```

Comment:
you forgot to include `video-interfaces.h` ?

Reply:
Fixed.